Demo/UI Tests/ThreeDSecure UI Tests/ThreeDSecure_UITests_Extensions.swift

```diff
@@ -68,5 +68,5 @@ internal extension XCUIApplication {
 }
 
 internal extension TimeInterval {
-    static let threeDSecureTimeout = 20.0
+    static let threeDSecureTimeout = 30.0
```

Comment:
Some of the UI tests were timing out while waiting for elements in the Cardinal window - bumping timeouts as this UI element is fully out of our control

Reply:
The weird thing is that our UI and Integration test were relatively stable for the longest time.

These issues seem to be correlated to [our bump to Xcode 14.3 and Mac OS 13](https://github.com/braintree/braintree_ios/pull/1031/files) ~2 months ago. We should dig in GH Actions Issues to see if other users are experience similar issues.

Reply:
The integration tests were stable but the UI tests have been flakey for a while - though they have certainly gotten worse. I looked through the runner GitHub issues and was unable to find something specific to this Xcode + macOS version combo, there are similar reports in earlier macOS and Xcode versions but ours were more consistent on those older versions.